.github/workflows/docker-build.yml

```diff
@@ -1,23 +1,14 @@
-name: Build Docker Container
+name: docker build
 
 on:
   workflow_call:
 
 jobs:
-  build:
+  docker-build:
     runs-on: ubuntu-latest
     steps:
-    - name: Free up the build space
-      uses: easimon/maximize-build-space@master
-      with:
-        root-reserve-mb: 30000
-        swap-size-mb: 1024
-        remove-dotnet: 'true'
-        remove-android: 'true'
-        remove-codeql: 'true'
-        remove-haskell: 'true'
```

Comment:
Could you re-add this block of code? This is needed since the docker container is so big, we are required to remove any unnecessary packages in github runner so we can build the docker container.

Reply:
ddone